lib/configuration.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, args):
         args.add_argument("-garmin_enable_upload",help="True will try to upload activities to Garmin", dest="garmin_enable_upload", type=str, default=os.environ.get('P2G_GARMIN_ENABLE_UPLOAD'))
         args.add_argument("-path",help="Path to output directory",dest="output_dir",type=str, default=os.environ.get('P2G_PATH'))
         args.add_argument("-num",help="Number of activities to download",dest="num_to_download",type=int, default=os.environ.get('P2G_NUM'))
+        args.add_argument("-workout_types",help="Process specified activity types only", dest="workout_types", type=str, default=os.environ.get('P2G_WORKOUT_TYPES'))
```

Comment:
Great idea for a feature, can you update the readme [Config ](https://github.com/philosowaffle/peloton-to-garmin#configuration) table aswell?

Reply:
"Done and dusted."